extensions/wikia/Discussions/maintenance/StaffWelcomePoster.class.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class StaffWelcomePoster {
+
+	const LANG_TO_STAFF_MAP = [
+		'de' => 24961680, // ForestFairy
+		'en' => 26339491, // Mira Laime
+		'es' => 12648798, // Luchofigo85
+		'fr' => 26442523, // Hypsoline
+		'it' => 3279487,  // Leviathan_89
+		'ja' => 29395778, // Kuro0222
+		'ko' => 24883131, // Miri-Nae
+		'nl' => 4142476,  // Yatalu
+		'pl' => 1117661,  // Nanaki
+		'pt' => 24005296, // Macherie ana
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)